weasyprint/__init__.py

```diff
@@ -75,6 +77,7 @@
     'full_fonts': False,
     'hinting': False,
     'cache': None,
+    'timeout': None
```

Comment:
Is that required? I think that `timeout` should just be a CLI option, but not something available in the Python API.

Reply:
I added it as it is also used by CLI parser for default options but I guess it doesn't matter as anyway the value is `None`. Will remove it.